MSAL/src/native_auth/controllers/credentials/MSALNativeAuthCredentialsController.swift

```diff
@@ -88,7 +88,7 @@ final class MSALNativeAuthCredentialsController: MSALNativeAuthTokenController,
                       userAccountResult: MSALNativeAuthUserAccountResult) async -> RefreshTokenCredentialControllerResponse {
         MSALLogger.log(level: .verbose, context: context, format: "Refresh started")
         let telemetryEvent = makeAndStartTelemetryEvent(id: .telemetryApiIdRefreshToken, context: context)
-        let scopes = authTokens.accessToken.scopes.array as? [String] ?? []
+        let scopes = authTokens.accessToken.scopes?.array as? [String] ?? []
```

Comment:
Why is this nullable (given all the conversations we've had about not allowing null and empty scopes)?